client/src/components/new-request/search.vue

```diff
@@ -1,17 +1,7 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal">
     <v-row justify="end">
-      <v-col cols="6" class="copy-normal">I need a name to:</v-col>
-      <v-col cols="6" style="display: flex; justify-content: flex-end">
-        <business-structures-tool>
-          <template #start="{ homeModal }">
-             <div id="#help-me-choose-activator"
-                  class="link-std"
-                  style="background-color: white"
-                  @click="homeModal()">Help Me Choose</div>
-          </template>
-        </business-structures-tool>
-      </v-col>
+      <v-col cols="12" class="copy-normal">I need a name to:</v-col>
```

Comment:
I changed this col to 12 units wide so it's always on one line (otherwise the responsive layout is messed up).